pkgs/top-level/perl-packages.nix

```diff
@@ -16575,6 +16575,7 @@ let
       url = mirror://cpan/authors/id/L/LD/LDACHARY/Text-Unaccent-1.08.tar.gz;
       sha256 = "0avk50kia78kxryh2whmaj5l18q2wvmkdyqyjsf6kwr4kgy6x3i7";
     };
+    patches = [ ../development/perl-modules/Text-Unaccent-vprintf.patch ];
```

Comment:
You could also set this without a patch: `NIX_CFLAGS_COMPILE = [ "-DHAS_VPRINTF" ];`

Reply:
good call; done in 41f17aab20c.